src/toil_scripts/gatk_germline/vqsr.py

```diff
@@ -0,0 +1,300 @@
+               '-recalFile', 'HAPSNP.recal',
+               '-tranchesFile', 'HAPSNP.tranches',
+               '-rscriptFile', 'HAPSNP.plots']
+
+    if config['unsafe_mode']:
+        command = ['-U', 'ALLOW_SEQ_DICT_INCOMPATIBILITY'] + command
+
+    outputs = {'HAPSNP.recal': None, 'HAPSNP.tranches': None, 'HAPSNP.plots': None}
+    docker_call(work_dir = work_dir,
+                env={'_JAVA_OPTIONS':'-Djava.io.tmpdir=/data/ -Xmx{}'.format(config['xmx'])},
+                parameters = command,
+                tool ='quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
+                inputs=inputs.keys(),
+                outputs=outputs,
+                mock=config['mock_mode'])
+
+    recal_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'HAPSNP.recal'))
+    tranches_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'HAPSNP.tranches'))
+    plots_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'HAPSNP.plots'))
+    return recal_id, tranches_id, plots_id
+
+
+def gatk_apply_variant_recalibration_snp(job, vcf_id, recal_id, tranches_id, config):
+    """
+    Apply variant quality score recalibration for SNP variants.
+
+    :param job: Job instance
+    :param vcf_id str: vcf file store ID
+    :param recal_id: recalibration table file store ID
+    :param tranches_id: tranches file store ID
+    :param config dict: pipeline configuration options and shared files
+    :return str: SNP recalibrated VCF file store ID
+    """
+    job.fileStore.logToMaster('Running GATK ApplyRecalibration (SNP Mode): {}'.format(config['uuid']))
+    work_dir = job.fileStore.getLocalTempDir()
+    references = ['genome.fa', 'genome.fa.fai', 'genome.dict']
+    inputs = {'toil.vcf': vcf_id,
+              'HAPSNP.tranches': tranches_id,
+              'HAPSNP.recal': recal_id}
+    inputs.update({key: config[key] for key in references})
+    get_files_from_filestore(job, work_dir, inputs)
+
+    command = ['-T', 'ApplyRecalibration',
+               '-input', 'toil.vcf',
+               '-o', 'toil.vqsr.vcf',
+               '-R', 'genome.fa',
+               '-nt', '1',
+               '-ts_filter_level', '99.0',
+               '-tranchesFile', 'HAPSNP.tranches',
+               '-recalFile', 'HAPSNP.recal',
+               '-mode', 'SNP']
+
+    if config['unsafe_mode']:
+        command = ['-U', 'ALLOW_SEQ_DICT_INCOMPATIBILITY'] + command
+
+    outputs={'toil.vqsr.vcf': None}
+    docker_call(work_dir = work_dir,
+                env={'_JAVA_OPTIONS':'-Djava.io.tmpdir=/data/ -Xmx{}'.format(config['xmx'])},
+                parameters = command,
+                tool = 'quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
+                inputs=inputs.keys(),
+                outputs=outputs,
+                mock=config['mock_mode'])
+    return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'toil.vqsr.vcf'))
+
+
+def gatk_variant_recalibrator_indel(job, vcf_id, config):
```

Comment:
How much can we factor out between this and `gatk_variant_recalibrator_snp`? They're 99% the same.